x-pack/plugins/enterprise_search/public/applications/workplace_search/routes.ts

```diff
@@ -6,8 +6,7 @@
 
 import { generatePath } from 'react-router-dom';
 
-import { CURRENT_MAJOR_VERSION } from '../../../common/version';
-import { ENT_SEARCH_DOCS_PREFIX } from '../shared/constants';
+import { docLinksService } from '../shared/doc_links/doc_links';
```

Comment:
nit: just import directly from the folder index.ts

```suggestion
import { docLinksService } from '../shared/doc_links'; 
```